activities/member-relations/index.md

```diff
@@ -2,8 +2,10 @@
 type: Index
 ---
 
-These are all the activities, documents and templates for member relations:
+These are all the activities, documents, guides and templates for member relations:
 
+## Resources:
```

Comment:
Level 2 heading where is no level 1 heading is not valid markdown, so remove if no heading.

I would actually remove since there is no other type of content, and the separation might not make sense when the need arises.